src/test/scala/org/broadinstitute/dsde/rawls/webservice/EntityApiServiceSpec.scala

```diff
@@ -315,22 +315,8 @@ class EntityApiServiceSpec extends ApiServiceSpec {
       }
   }
 
-  it should "return 204 on entity delete" in withTestDataApiServices { services =>
-    Delete(s"/workspaces/${testData.workspace.namespace}/${testData.workspace.name}/entities/${testData.sample8.entityType}/${testData.sample8.name}") ~>
-      sealRoute(services.entityRoutes) ~>
-      check {
-        assertResult(StatusCodes.NoContent) {
-          status
-        }
-        assertResult(None) {
-          runAndWait(entityQuery.get(SlickWorkspaceContext(testData.workspace), testData.sample8.entityType, testData.sample8.name))
-        }
-      }
-  }
-
```

Comment:
I removed this test because it was added last week as a simplified case of the test that follows this one. Since we don't have any delete for the time being, this test is now pointless.